mne/io/brainvision/brainvision.py

```diff
@@ -33,9 +33,10 @@ class RawBrainVision(BaseRaw):
     ----------
     vhdr_fname : str
         Path to the EEG header file.
-    montage : str | None | instance of Montage
-        Path or instance of montage containing electrode positions.
-        If None, sensor locations are (0,0,0). See the documentation of
+    montage : str | True | None | instance of Montage
+        Path or instance of montage containing electrode positions. If
+        True, read embedded sensor locations. If None, sensor locations
+        are (0,0,0). See the documentation of
         :func:`mne.channels.read_montage` for more information.
```

Comment:
> if I implement the above it will break the brainvision test at line 297 since now it will read electrode position info by default thus requiring the user to explicitly set channels without coordinates (or rather with 0,0,0) to either eog or misc.

This is bad because it sounds like it can break people's existing pipelines.

We could make `None` only read + use if this problem doesn't crop up, but that's starting to seem a bit too magical / fragile.

Perhaps this is why `None` does nothing on `master` and `True` is available.